packages/evm/.soliumrc.json

```diff
@@ -16,7 +16,7 @@
     "lbrace": "error",
     "mixedcase": "error",
     "camelcase": "error",
-    "uppercase": "error",
+    "uppercase": "off",
```

Comment:
Fixes the linting issue with `string public constant name`. I don't think this rule is that useful anyway :)